Mu2eUtilities/src/stoppedMuonList.cc

```diff
@@ -0,0 +1,38 @@
+#include "Mu2eUtilities/inc/stoppedMuonList.hh"
+
+namespace mu2e {
+
+  //================================================================
+  namespace {
+
+    template<class H>
+    std::vector<art::Ptr<SimParticle> > stoppedMuonListImpl(H simh) {
+
+      std::vector<art::Ptr<SimParticle> > res;
+
+      const auto& sims = *simh;
+
+      for(auto i = sims.begin(); i != sims.end(); ++i) {
+        const auto& inpart = i->second;
+        if((inpart.pdgId() == PDGCode::mu_minus) &&
```

Comment:
We will eventually need to process stopped mu+, stopped pi-, etc.  is the idea to have dedicated functions for each?  If so, the name stoppedMuonList is ambiguous.  I think a more extensible design is a generic stoppedParticleList function which takes pdgcode and process code as arguments.  Or make this a functor, and provide the pdgcode and process on construction.